### ISSUE TITLE
Cannot run wallaby on vitest typescript project (ReferenceError: $_$wp is not defined​​)

### ISSUE THREAD

**Issue by mlewando** [wallabyjs/public#3297]
### Issue description or question

I'm trying to run wallaby on my project that uses vitest for test. Tests works correctly on console, but wallaby is failing to start due to `ReferenceError: $_$wp is not defined​​` error

### Wallaby diagnostics report

```
{
  editorVersion: '1.84.2',
  pluginVersion: '1.0.371',
  editorType: 'VSCode',
  osVersion: 'darwin 23.0.0',
  nodeVersion: 'v18.16.1',
  coreVersion: '1.0.1437',
  checksum: 'ZDEzM2QzNmRhZjJiODUzOGFhMTY4OTc2MjI1MWIwYjksMTY4ODUxNTIwMDAwMCww',
  config: {
    diagnostics: {
      vitest: {
        file: { config: "import { defineConfig } from 'vite';\nimport react from '@vitejs/plugin-react-swc';\n\nexport default defineConfig({\n  plugins: [react()],\n});\n" },
        config: {
          allowOnly: true,
          watch: true,
          globals: false,
          environment: 'node',
          threads: true,
          clearMocks: false,
          restoreMocks: false,
          mockReset: false,
          include: [ '**/*.{test,spec}.?(c|m)[jt]s?(x)' ],
          exclude: [
            '**/node_modules/**',
            '**/dist/**',
            '**/cypress/**',
            '**/.{idea,git,cache,output,temp}/**',
            '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*'
          ],
          testTimeout: 5000,
          hookTimeout: 10000,
          teardownTimeout: 10000,
          isolate: true,
          watchExclude: [ '**/node_modules/**', '**/dist/**' ],
          forceRerunTriggers: [ '**/package.json/**', '**/{vitest,vite}.config.*/**' ],
          update: false,
          reporters: [ 'default' ],
          silent: false,
          hideSkippedTests: false,
          api: { middlewareMode: true },
          ui: false,
          uiBase: '/__vitest__/',
          open: true,
          css: { include: [], modules: { classNameStrategy: 'stable' } },
          coverage: {
            provider: 'v8',
            enabled: false,
            clean: true,
            cleanOnRerun: true,
            reportsDirectory: './coverage',
            exclude: [
              'coverage/**',
              'dist/**',
              'packages/*/test?(s)/**',
              '**/*.d.ts',
              '**/virtual:*',
              '**/__x00__*',
              '**/\x00*',
              'cypress/**',
              'test?(s)/**',
              'test?(-*).?(c|m)[jt]s?(x)',
              '**/*{.,-}{test,spec}.?(c|m)[jt]s?(x)',
              '**/__tests__/**',
              '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*',
              '**/vitest.{workspace,projects}.[jt]s?(on)',
              '**/.{eslint,mocha,prettier}rc.{?(c|m)js,yml}'
            ],
            reportOnFailure: false,
            reporter: [ [ 'text', {} ], [ 'html', {} ], [ 'clover', {} ], [ 'json', {} ] ],
            extension: [
              '.js',  '.cjs',
              '.mjs', '.ts',
              '.mts', '.cts',
              '.tsx', '.jsx',
              '.vue', '.svelte'
            ],
            allowExternal: false
          },
          fakeTimers: { loopLimit: 10000, shouldClearNativeTimers: true, toFake: [ 'setTimeout', 'clearTimeout', 'setInterval', 'clearInterval', 'setImmediate', 'clearImmediate', 'Date' ] },
          maxConcurrency: 5,
          dangerouslyIgnoreUnhandledErrors: false,
          typecheck: {
            checker: 'tsc',
            include: [ '**/*.{test,spec}-d.?(c|m)[jt]s?(x)' ],
            exclude: [
              '**/node_modules/**',
              '**/dist/**',
              '**/cypress/**',
              '**/.{idea,git,cache,output,temp}/**',
              '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*'
            ]
          },
          slowTestThreshold: 300,
          config: '<homeDir>/Projects/split/split-web/vite.config.ts',
          defines: {},
          root: '<homeDir>/Projects/split/split-web',
          mode: [],
          inspect: false,
          inspectBrk: false,
          singleThread: false,
          deps: {
            moduleDirectories: [ '/node_modules/' ],
            optimizer: { ssr: { enabled: true }, web: { enabled: true } },
            web: { transformAssets: true, transformCss: true, transformGlobPattern: [] }
          },
          server: { deps: { inline: [ {}, {}, {}, '@nuxt/test-utils' ], moduleDirectories: [ '/node_modules/' ], cacheDir: 'node_modules/.vitest' } },
          snapshotOptions: { snapshotFormat: {}, updateSnapshot: 'new', snapshotEnvironment: null },
          experimentalVmWorkerMemoryLimit: 6871947673,
          setupFiles: [],
          cache: { dir: '<homeDir>/Projects/split/split-web/node_modules/.vitest' },
          sequence: { hooks: 'parallel' },
          environmentMatchGlobs: [],
          browser: { enabled: false, headless: false, slowHijackESM: true, api: { middlewareMode: true } },
          testTransformMode: {},
          package: {
            version: '0.34.6',
            urls: { hooks: 'file://<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/runners/node/hooks.mjs' },
            paths: { root: '<homeDir>/Projects/split/split-web/node_modules/vitest', dist: '<homeDir>/Projects/split/split-web/node_modules/vitest/dist' }
          }
        }
      }
    },
    testFramework: { version: 'vitest@0.14.0', configurator: 'vitest@0.14.0', reporter: 'vitest@0.14.0', starter: 'vitest@0.14.0', autoDetected: true },
    preserveComments: false,
    extractComments: true,
    files: [
      { pattern: '**/*.{test,spec}.?(c|m)[jt]s?(x)', ignore: true, trigger: true, load: true, file: true },
      { pattern: '**/node_modules/**', ignore: true, trigger: true, load: true, file: true, test: true },
      { pattern: '**/dist/**', ignore: true, trigger: true, load: true, file: true, test: true },
      { pattern: '**/cypress/**', ignore: true, trigger: true, load: true, file: true, test: true },
      { pattern: '**/.{idea,git,cache,output,temp}/**', ignore: true, trigger: true, load: true, file: true, test: true },
      { pattern: '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*', ignore: true, trigger: true, load: true, file: true, test: true },
      { pattern: '**/*.*', ignore: false, trigger: true, load: true, order: 1 }
    ],
    tests: [
      { pattern: '**/node_modules/**', ignore: true, trigger: true, load: true, test: true, file: false },
      { pattern: '**/dist/**', ignore: true, trigger: true, load: true, test: true, file: false },
      { pattern: '**/cypress/**', ignore: true, trigger: true, load: true, test: true, file: false },
      { pattern: '**/.{idea,git,cache,output,temp}/**', ignore: true, trigger: true, load: true, test: true, file: false },
      { pattern: '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*', ignore: true, trigger: true, load: true, test: true, file: false },
      { pattern: '**/*.{test,spec}.?(c|m)[jt]s?(x)', ignore: false, trigger: true, load: true, test: true, order: 2 }
    ],
    workers: { initial: 1, regular: 1, recycle: false },
    captureConsoleLog: true,
    filesWithNoCoverageCalculated: [],
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    compilers: {},
    logLimits: { inline: { depth: 5, elements: 5000 }, values: { default: { stringLength: 200 }, autoExpand: { elements: 5000, stringLength: 8192, depth: 10 } } },
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore|c8 ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    mapConsoleMessagesStackTrace: false,
    extensions: {},
    env: {
      type: 'node',
      params: { runner: '--experimental-loader=file://<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/runners/node/hooks.mjs' },
      runner: '<homeDir>/.nvm/versions/node/v18.16.1/bin/node',
      viewportSize: { width: 800, height: 600 },
      options: { width: 800, height: 600 },
      bundle: true
    },
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    runAllTestsWhenNoAffectedTests: true,
    symlinkNodeModules: true,
    configCode: 'auto.detect#2145583041'
  },
  packageJSON: {
    dependencies: { firebase: '^10.5.2', react: '^18.2.0', 'react-dom': '^18.2.0' },
    devDependencies: {
      '@types/react': '^18.2.35',
      '@types/react-dom': '^18.2.14',
      '@typescript-eslint/eslint-plugin': '^6.9.1',
      '@typescript-eslint/parser': '^6.9.1',
      '@vitejs/plugin-react-swc': '^3.4.1',
      '@vitest/runner': '^0.34.6',
      autoprefixer: '^10.4.16',
      cssnano: '^6.0.1',
      eslint: '^8.53.0',
      'eslint-config-prettier': '^9.0.0',
      'eslint-plugin-import': '^2.29.0',
      'eslint-plugin-react': '^7.33.2',
      'eslint-plugin-react-hooks': '^4.6.0',
      'eslint-plugin-react-refresh': '^0.4.4',
      'firebase-tools': '^12.8.0',
      postcss: '^8.4.31',
      prettier: '^3.0.3',
      'prettier-plugin-tailwindcss': '^0.5.6',
      tailwindcss: '^3.3.5',
      typescript: '^5.2.2',
      vite: '^4.5.0',
      'vite-node': '^0.34.6',
      vitest: '^0.34.6'
    }
  },
  fs: { numberOfFiles: 24 },
  debug: [
    '2023-11-17T07:00:16.060Z config Attempting automatic configuration for angular\n',
    '2023-11-17T07:00:16.062Z angular/cli config Angular CLI not found.\n',
    '2023-11-17T07:00:16.062Z config Finished attempting automatic configuration for angular (2ms)\n',
    '2023-11-17T07:00:16.062Z config Attempting automatic configuration for jest\n',
    "2023-11-17T07:00:16.063Z jest/config Error: Module jest-cli is not found in '<homeDir>/Projects/split/split-web'.\n" +
      '    at Object.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:32:5336)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:31:20403)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:31:19698)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:31:19607\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:31:19401)\n' +
      '    at Object.loadJest (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:32:2934)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:19:30355\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:20:3022)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:20:2317)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:20:2226\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:20:2020)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:19:30043)\n' +
      '    at Object.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:20:1697)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:20:3022)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:20:2317)\n' +
      '    at <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:20:2226\n' +
      '    at new Promise (<anonymous>)\n' +
      '    at s (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:20:2020)\n' +
      '    at Object.n [as configure] (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:20:1578)\n' +
      '    at Config.<anonymous> (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:15:24268)\n' +
      '    at r (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:15:15489)\n' +
      '    at Object.next (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:15:14784)\n' +
      '    at o (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/server.js:15:14533)\n',
    '2023-11-17T07:00:16.063Z config Finished attempting automatic configuration for jest (1ms)\n',
    '2023-11-17T07:00:16.063Z config Attempting automatic configuration for vitest\n',
    '2023-11-17T07:00:16.065Z vitest/config Detected Vitest (0.34.6).\n',
    '2023-11-17T07:00:16.396Z config Finished attempting automatic configuration for vitest (333ms)\n',
    '2023-11-17T07:00:16.397Z project Wallaby Node version: v18.16.1\n',
    '2023-11-17T07:00:16.397Z project Wallaby config: <homeDir>/Projects/split/split-web/auto.detect\n',
    '2023-11-17T07:00:16.418Z fs File system starting\n',
    '2023-11-17T07:00:16.435Z fs File system scan completed\n',
    '2023-11-17T07:00:16.438Z project File cache: <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/projects/86ef1013cb42f462\n',
    '2023-11-17T07:00:16.465Z uiService Listening port 51235\n',
    '2023-11-17T07:00:16.471Z workers Parallelism for initial run: 1, for regular run: 1\n',
    '2023-11-17T07:00:16.471Z workers Starting run worker instance #0\n',
    '2023-11-17T07:00:16.471Z workers Web server is listening at 49840\n',
    '2023-11-17T07:00:16.473Z project Stopping process pool\n',
    '2023-11-17T07:00:16.473Z project File cache is up-to-date, starting full test run\n',
    '2023-11-17T07:00:16.473Z project Test run started; run priority: 3\n',
    '2023-11-17T07:00:16.473Z project Running all tests\n',
    '2023-11-17T07:00:16.474Z workers Starting test run, priority: 3\n',
    '2023-11-17T07:00:16.474Z nodeRunner Starting sandbox [worker #0, session #wng31]\n',
    '2023-11-17T07:00:16.474Z nodeRunner Preparing sandbox [worker #0, session #wng31]\n',
    '2023-11-17T07:00:16.568Z workers Started run worker instance (delayed) #0\n',
    '2023-11-17T07:00:16.568Z nodeRunner Prepared sandbox [worker #0, session #wng31]\n',
    '2023-11-17T07:00:16.568Z workers [worker #0, session #wng31] Running tests in sandbox\n',
    '2023-11-17T07:00:16.873Z workers Scheduling Vitest Run (wng31): 2023-11-17T07:00:16.744Z\n',
    '2023-11-17T07:00:16.937Z workers Sandbox (active) [wng31] error: ReferenceError: $_$wp is not defined\n' +
      '    at ./src/splitAmountByAdjustments.test.ts:1:12\n' +
      '    at VitestExecutor.runModule (file://./node_modules/vite-node/dist/client.mjs:342:11)\n' +
      '    at VitestExecutor.runModule (file://./node_modules/vitest/dist/vendor-execute.07d1a420.js:1150:20)\n' +
      '    at VitestExecutor.directRequest (file://./node_modules/vite-node/dist/client.mjs:326:16)\n' +
      '    at VitestExecutor.cachedRequest (file://./node_modules/vite-node/dist/client.mjs:189:14)\n' +
      '    at VitestExecutor.executeId (file://./node_modules/vite-node/dist/client.mjs:165:12)\n' +
      '    at collectTests (file://./node_modules/@vitest/runner/dist/index.js:511:7)\n' +
      '    at startTests (file://./node_modules/@vitest/runner/dist/index.js:840:17)\n' +
      '    at file://./node_modules/vitest/dist/entry.js:103:7\n' +
      '    at withEnv (file://./node_modules/vitest/dist/entry.js:73:5)\n',
    "2023-11-17T07:00:16.947Z workers Sandbox (active) [wng31] error: TypeError: Cannot read properties of undefined (reading 'error')\n" +
      '    at runFiles (file://./node_modules/vitest/dist/vendor-node.a7c48fe1.js:7330:33)\n' +
      '    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n' +
      '    at async Promise.allSettled (index 0)\n' +
      '    at async Object.runTests (file://./node_modules/vitest/dist/vendor-node.a7c48fe1.js:7372:27)\n' +
      '    at async Promise.all (index 1)\n' +
      '    at async Object.runTests (file://./node_modules/vitest/dist/vendor-node.a7c48fe1.js:7664:5)\n' +
      '    at async file://./node_modules/vitest/dist/vendor-node.a7c48fe1.js:11206:9\n' +
      '    at async Vitest.runFiles (file://./node_modules/vitest/dist/vendor-node.a7c48fe1.js:11220:12)\n',
    '2023-11-17T07:00:16.952Z workers Vitest Run Complete (wng31): 2023-11-17T07:00:16.947Z\n',
    '2023-11-17T07:00:16.953Z fs No metadata for added file found: node_modules/.vitest/results.json\n',
    '2023-11-17T07:00:16.953Z workers [wng31] Run 0 test(s), skipped 0 test(s)\n',
    '2023-11-17T07:00:16.954Z workers [wng31] Sandbox is responsive, closing it\n',
    "2023-11-17T07:00:16.955Z workers Failed to map the stack to user code, entry message: TypeError: Cannot read properties of undefined (reading 'error'), stack: TypeError: Cannot read properties of undefined (reading 'error')\n" +
      '    at Object.runResult (<homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/runners/node/vitest@0.14.0/initializer.js:14:19794)\n' +
      '    at runFiles (file://<homeDir>/Projects/split/split-web/node_modules/vitest/dist/vendor-node.a7c48fe1.js:7330:33)\n' +
      '    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n' +
      '    at async Promise.allSettled (index 0)\n' +
      '    at async Object.runTests (file://<homeDir>/Projects/split/split-web/node_modules/vitest/dist/vendor-node.a7c48fe1.js:7372:27)\n' +
      '    at async Promise.all (index 1)\n' +
      '    at async Object.runTests (file://<homeDir>/Projects/split/split-web/node_modules/vitest/dist/vendor-node.a7c48fe1.js:7664:5)\n' +
      '    at async file://<homeDir>/Projects/split/split-web/node_modules/vitest/dist/vendor-node.a7c48fe1.js:11206:9\n' +
      '    at async Vitest.runFiles (file://<homeDir>/Projects/split/s\n',
    '2023-11-17T07:00:16.955Z project Test run finished\n',
    '2023-11-17T07:00:16.955Z project Processed console.log entries\n',
    '2023-11-17T07:00:16.955Z project Processed loading sequences\n',
    '2023-11-17T07:00:16.955Z project Processed executed tests\n',
    '2023-11-17T07:00:16.956Z project Processed code coverage\n',
    '2023-11-17T07:00:16.959Z project Test run result processed and sent to IDE\n'
  ]
}
```

and this is the output from wallaby console:

```
‍​[Info]​ Started Wallaby.js Core v1.0.1437
​​[Error] Runtime error: ReferenceError: $_$wp is not defined​​
​​[Error]     at ./src/splitAmountByAdjustments.test.ts:1:12​​
​​[Error]     at VitestExecutor.runModule (file://./node_modules/vite-node/dist/client.mjs:342:11)​​
​​[Error]     at VitestExecutor.runModule (file://./node_modules/vitest/dist/vendor-execute.07d1a420.js:1150:20)​​
​​[Error]     at VitestExecutor.directRequest (file://./node_modules/vite-node/dist/client.mjs:326:16)​​
​​[Error]     at VitestExecutor.cachedRequest (file://./node_modules/vite-node/dist/client.mjs:189:14)​​
​​[Error]     at VitestExecutor.executeId (file://./node_modules/vite-node/dist/client.mjs:165:12)​​
​​[Error]     at collectTests (file://./node_modules/@vitest/runner/dist/index.js:511:7)​​
​​[Error]     at startTests (file://./node_modules/@vitest/runner/dist/index.js:840:17)​​
​​[Error]     at file://./node_modules/vitest/dist/entry.js:103:7​​
​​[Error]     at withEnv (file://./node_modules/vitest/dist/entry.js:73:5)​​
​​[Error] Runtime error: TypeError: Cannot read properties of undefined (reading 'error')​​
​​[Error]     at runFiles (file://./node_modules/vitest/dist/vendor-node.a7c48fe1.js:7330:33)​​
​​[Error]     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)​​
​​[Error]     at async Promise.allSettled (index 0)​​
​​[Error]     at async Object.runTests (file://./node_modules/vitest/dist/vendor-node.a7c48fe1.js:7372:27)​​
​​[Error]     at async Promise.all (index 1)​​
​​[Error]     at async Object.runTests (file://./node_modules/vitest/dist/vendor-node.a7c48fe1.js:7664:5)​​
​​[Error]     at async file://./node_modules/vitest/dist/vendor-node.a7c48fe1.js:11206:9​​
​​[Error]     at async Vitest.runFiles (file://./node_modules/vitest/dist/vendor-node.a7c48fe1.js:11220:12)​​
​[Info]​ console.error: ⎯⎯⎯⎯⎯⎯ Failed Suites 1 ⎯⎯⎯⎯⎯⎯⎯
​[Info]​ console.error: 
​[Info]​ console.error:  FAIL  src/splitAmountByAdjustments.test.ts [ src/splitAmountByAdjustments.test.ts ]
​[Info]​ console.error: ReferenceError: $_$wp is not defined
​[Info]​ console.error:  ❯ src/splitAmountByAdjustments.test.ts:1:12
​[Info]​ console.error: 
​[Info]​ console.error: ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯
​[Info]​ console.error: 
​[Info]​ ⎯⎯⎯⎯⎯⎯ Unhandled Error ⎯⎯⎯⎯⎯⎯⎯
​[Info]​ console.error: TypeError: Cannot read properties of undefined (reading 'error')
​[Info]​ console.error:  ❯ Object.runResult ../../../.vscode/extensions/wallabyjs.wallaby-vscode-1.0.371/wallabya7c4ae/runners/node/vitest@0.14.0/initializer.js:14:19794
​[Info]​ console.error:  ❯ runFiles node_modules/vitest/dist/vendor-node.a7c48fe1.js:7330:33
​[Info]​ console.error:  ❯ process.processTicksAndRejections node:internal/process/task_queues:95:5
​[Info]​ console.error:  ❯ Object.runTests node_modules/vitest/dist/vendor-node.a7c48fe1.js:7372:27
​[Info]​ console.error:  ❯ Object.runTests node_modules/vitest/dist/vendor-node.a7c48fe1.js:7664:5
​[Info]​ console.error:  ❯ async file:/Users/mateuszlewandowski/Projects/split/split-web/node_modules/vitest/dist/vendor-node.a7c48fe1.js:11206:9
​[Info]​ console.error:  ❯ Vitest.runFiles node_modules/vitest/dist/vendor-node.a7c48fe1.js:11220:12
​[Info]​ console.error: 
​[Info]​ console.error: 
```

**Comment by smcenlly** [comment]
We tried creating a sample project with your dependencies and tests, and everything is working for us.

The version of Wallaby that you are using was released 4.5 months ago, and there have been a number of breaking changes to `vitest` since then that have required updates to Wallaby; we think your problem is a result of an incompatible `vitest` version.

Can you please update Wallaby core to the latest version, we think it will fix your problem. If your license has expired, you may [request a trial license](https://wallabyjs.com/download/#try-it-free) to confirm it fixes your issue.

**Comment by mlewando** [comment]
just to confirm - new version and it works fine :)
